src/app/(pages)/docs/[topic]/[doc]/page.tsx

```diff
@@ -11,6 +12,14 @@ const Doc = async ({ params }) => {
   const doc = await getDoc({ topic, doc: docSlug })
   const topics = await getTopics()
 
+  const helpfulThreads = await fetchHelpfulThreads()
```

Comment:
We are querying for these every time a docs page is loaded, but then filtering out most of the results?

I don't do much with nextjs but isn't this supposed to be in a `getStaticProps` function?